pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -17,13 +17,27 @@ limitations under the License.
 package enqueue
 
 import (
+	"fmt"
+	"strconv"
+	"strings"
+
 	"github.com/golang/glog"
 
 	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/conf"
 	"volcano.sh/volcano/pkg/scheduler/framework"
 	"volcano.sh/volcano/pkg/scheduler/util"
 )
 
+// IdleResMultiplierKey
+const IdleResMultiplierKey = "idleres-mul"
```

Comment:
use camel case for variable

This is the idle resource over occupy parameter.  How about calling overcommitment？

Reply:
so shall i make variable name as overcommitmentKey & overcommitmentVal .
const overcommitmentKey = "overcommitment-mul"

Reply:
const idleResMultiplierKey = "overcommitment-mem-factor" is this okay?